src/Idris/Package/Parser.hs

```diff
@@ -84,7 +68,7 @@ filename = (do
                 -- This also moves away from tying filenames to identifiers, so
                 -- it will also accept hyphens
                 -- (https://github.com/idris-lang/Idris-dev/issues/2721)
-    filename <- P.stringLiteral <* someSpace'
+    filename <- fst <$> stringLiteral
```

Comment:
Is there any reason you removed the space afterwards?

Reply:
This is now a call to the `stringLiteral` in `Idris.Parser.Helpers` instead of the one in `Text.Parser.Char`.

The new one uses `tokenFC` to consume trailing space with `whiteSpace`, which is defined as `someSpace <|> pure ()`.  It changes the semantics slightly in that it used to *require* space after the string, and now it merely accepts it.  But I think this is right.

Reply:
Yeah, that is fine